python/veles/objects/chunk.py

```diff
@@ -15,6 +19,15 @@ def __init__(self, client, proto_obj=None, id_path=None, parent=None):
         self._end = proto_obj.chunk_end
         self._blob = parent.get_blob() if parent is not None else None
         self._cursor = self._start
+        self.items = [ChunkDataItem(item) for item in self._proto_obj.items]
+
+    def _from_another(self, other):
+        super(Chunk, self)._from_another(other)
+        self.items = other.items
+
+    def items_from_another(self, other):
```

Comment:
Why is _from_another private and items_from_another not? Also s/items_from_another/copy_items? Finally shouldn't this method do self.items = other.items[:] at the very least? Or maybe even copy.deepcopy?